cypress/e2e/views/assessment.view.ts

```diff
@@ -15,6 +15,8 @@ limitations under the License.
 */
 export const stakeholderSelect = "input[aria-label='stakeholder-select-toggle']";
```

Comment:
Hi @ibolton336 , Thanks for the PR . It doesn't look like this selector is being used . So, this needs to be removed .

Reply:
Np @nachandr. Looks like it is still being used here:

https://github.com/konveyor/tackle-ui-tests/blob/caf4bd89c1b068615df11c33b1f32efa6b51d7ad/cypress/e2e/tests/migration/applicationinventory/applications/interlinked.test.ts#L153